src/game/World.java

```diff
@@ -99,11 +102,18 @@ public void draw(Viewport vp) {
 		for (Entity e : this.backgroundsprites) {
 			e.draw(vp);
 		}
+
 		Shape view = vp.getGameViewShape();
 		Rectangle viewRect = new Rectangle(view.getMinX(), view.getMinY(),
 				view.getWidth(),
 				view.getHeight());
 
+		doSunLighting((int) viewRect.getX() - 10,
+				(int) (viewRect.getX() + view.getWidth()) + 10,
+				(int) viewRect.getY() - 10,
+				(int) (viewRect.getY() + view.getHeight()) + 10,
+				63);

```

Comment:
Maybe use constants instead of 10 and 63?

Reply:
Yeah okay